docs/live_migration.md

```diff
@@ -0,0 +1,162 @@
+# Live Migration
+
+This document gives two examples of how to use the live migration
+support in Cloud Hypervisor: 1) local migration - migrating between two
+VMs running on the same machine; 2) nested-vm migration - migrating
+between two nested VMs whose host VMs are running on the same machine.
+
+## Local Migration
+Launch the source VM with an additional disk device for testing purpose
+(on the host machine):
+```bash
+$ head -c 1M < /dev/urandom > tmp.img # create a dummy image for testing
+$ target/release/cloud-hypervisor
+    --kernel ~/workloads/vmlinux \
+    --pmem file=~/workloads/focal.raw,discard_writes=on \
```

Comment:
This example seems to be about testing the virtio-block heavy use. I think you should probably document an approach that is not trying to exercise the system.

Reply:
Sounds good. I removed the testing workload of virtio-block part to keep the 'local migration' example simple. 

Please let me know if you want me to simplify the  'nested-vm migration' example in the same way.